src/pseudopeople/noise_functions.py

```diff
@@ -6,30 +6,41 @@
 from vivarium import ConfigTree
 from vivarium.framework.randomness import RandomnessStream
 
+from pseudopeople import schema_entities
 from pseudopeople.configuration import Keys
 from pseudopeople.constants import paths
 from pseudopeople.data.fake_names import fake_first_names, fake_last_names
 from pseudopeople.utilities import get_index_to_noise, vectorized_choice
 
 
 def omit_rows(
+    form_name: str,
     form_data: pd.DataFrame,
     configuration: ConfigTree,
     randomness_stream: RandomnessStream,
 ) -> pd.DataFrame:
     """
-
+    Function that omits rows from a dataset and returns only the remaining rows.  Note that for the ACS and CPS forms
+      we need to account for oversampling in the PRL simulation so a helper function has been hadded here to do so.
+    :param form_name: Form object being noised
     :param form_data:  pd.DataFrame of one of the form types used in Pseudopeople
     :param configuration: ConfigTree object containing noise level values
     :param randomness_stream: RandomnessStream object to make random selection for noise
     :return: pd.DataFrame with rows from the original dataframe removed
     """
 
     noise_level = configuration.probability
-    to_noise_idx = get_index_to_noise(
-        form_data, noise_level, randomness_stream, "omission_choice"
+    # Account for ACS and CPS oversampling
+    if form_name in [schema_entities.FORMS.acs.name, schema_entities.FORMS.cps.name]:
+        noise_level = 0.5 + noise_level / 2
```

Comment:
@rmudambi Hmm, seems like we should add protection against a user trying to set omission to None (instead of 0)?

Reply:
Doesn't your type checking validation already do that?